Xit/FileView/CommitEntryController.swift

```diff
@@ -126,6 +136,18 @@ final class CommitEntryController: NSViewController,
     updateStagedStatus()
     updateCommitButton()
     amendChcekbox.boolValue = repoUIController?.isAmending ?? false
+
+    updateGuide()
+
+    kvObservers.append(defaults.observe(\.guideWidth) {
```

Comment:
I like that you made it update live. Please use the Combine publisher version like I did with `stripSink`.